portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,44 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <nav>
+      <label class="logo">KvngSnow's Portfolio</label>
+      <ul>
+          <li><a href="http://www.github.com/JalenJenkins">Github</a></li>
+          <li><a href="https://www.instagram.com/kvngsno">Instagram</a></li>
+          <li><a href="https://www.linkedin.com/in/jalen-jenkins-b75772186">Linkedin</a><li> 
+          </ul>
+
+    <div id="image"> <img src = "images/self image.jpg" alt = "A picture of myself" style="width:128px;height:128px;"></div>
```

Comment:
A smaller nit here, remove the extra spaces: around " = ", and before the " <img". I'd also suggest, since the line is pretty long, breaking this out into three lines, opening div, img, closing div.